packages/cli/src/commands/commerce/catalog/create.ts

```diff
@@ -0,0 +1,139 @@
+import {bold} from 'chalk';
+import type {
+  CreateCatalogConfigurationModel,
+  CatalogConfigurationModel,
+  CatalogFieldsMapping,
+  New,
+  PlatformClient,
+} from '@coveord/platform-client';
+import {Command, Flags} from '@oclif/core';
+import {
+  HasNecessaryCoveoPrivileges,
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import {Trackable} from '../../../lib/decorators/preconditions/trackable';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+
+export default class CatalogCreate extends Command {
+  public static description = `${bold.bgYellow(
+    '(alpha)'
+  )} Create a commerce catalog interactively`;
+
+  public static flags = {
+    // TODO: find an alternative to skip all the prompts without having too many flags.
+    withVariants: Flags.boolean({
+      char: 'v',
+      default: false,
+      helpGroup: 'Catalog structure',
+      description: 'Whether your catalog contains product variants',
+    }),
+    withAvailabilites: Flags.boolean({
+      char: 'a',
+      default: false,
+      helpGroup: 'Catalog structure',
+      description:
+        'Whether your catalog contains product location and/or availabilities',
+    }),
```

Comment:
can't we instead just do 'if there is at least one availability flag' ?

Reply:
not sure I get your proposition. How would you do that?

Reply:
`flags.availabilityFiles.length>0`

Reply:
Will address that in CDX-977